CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.0.0
```

Comment:
IDK? We've been on `beta` a few times now. In an earlier commit, I called this `beta4`. I can go back to that if you like.

What is our goal with 1.0?